tests/__tests__/api/user/getEmail.js

```diff
@@ -23,27 +23,25 @@ beforeEach(async () => {
   execSync("npm run reset-s", { stdio: "ignore" });
 });
 
-describe("Get User By Username Tests", () => {
-  test("get a user by a valid/existing username raisa", async () => {
-    const response = await request(app).get("/user/username/raisa");
+describe("Get User By Email Tests", () => {
+  test("get a user by a valid/existing email (Raisa)", async () => {
+    const response = await request(app).get("/user/email/raisa@raisa.com");
 
     showLog && console.log(response.body);
     expect(response.status).toBe(200);
-    expect(response.body).toStrictEqual(validUsernameQueryRaisa);
-    expect(response.body.username).toBe("raisa");
+    expect(response.body).toStrictEqual(validEmailQueryRaisa);
   });
 
-  test("get a user by a valid/existing username ethan", async () => {
-    const response = await request(app).get("/user/username/ethan");
+  test("get a user by a valid/existing email (Ethan)", async () => {
+    const response = await request(app).get("/user/email/ethan@ethan.com");
 
     showLog && console.log(response.body);
     expect(response.status).toBe(200);
-    expect(response.body).toStrictEqual(validUsernameQueryEthan);
-    expect(response.body.username).toBe("ethan");
+    expect(response.body).toStrictEqual(validEmailQueryEthan);
   });
 
-  test("attempt getting an invalid username", async () => {
-    const response = await request(app).get("/user/username/nonexistentuser");
+  test("attempt getting an invalid email", async () => {
+    const response = await request(app).get("/user/email/nonexistentuser");
```

Comment:
(n-b) you can make it clear that this `nonexistentuser` endpoint takes in an email address